web/themes/custom/pineapple/sass/layouts/views/_view--recipes.scss

```diff
@@ -0,0 +1,59 @@
+.view-recipes {
```

Comment:
The styling in this file isn't styling the views recipe page, it's styling the filters form on the views recipe page. Create a new folder that's form specific, rename this file and add it into there

Reply:
Done